snakePipes/shared/rules/ATAC.snakefile

```diff
@@ -15,10 +15,29 @@ rule filterFragments:
         --maxFragmentLength {params.cutoff}
         """
 
+# necessary for that MACS2 BAMPE fails, if there is just one fragment mapped
```

Comment:
Should we just add this to the alignmentSieve call? Then we only have to filter the file once. The process would be `run samtools idxstats and make a bed file of regions with 0 < coverage <= fragment_Count_cutoff`, then add that to the blacklist files given to `alignmentSieve`. I should note that I've never tested an empty blacklist file...

Reply:
Not sure, whether this works, because the blacklist needs to be produced after all the fragments were filtered from the BAM file. 

The approach I looked at was to produce a blacklist from the `filtered_bam/{sample}.filtered.bam` and use `alignmentSieve` to filter and blacklist in one run. The resulting bam file contained new low coverage scaffolds.